src/SFA.DAS.ApprenticeCommitments.Web/Pages/ConfirmYourApprenticeshipDetails.cshtml.cs

```diff
@@ -0,0 +1,16 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+
+namespace SFA.DAS.ApprenticeCommitments.Web.Pages
+{
+    public class ConfirmYourApprenticeshipDetailsModel : PageModel
+    {
+        public void OnGet()
+        {
+        }
+    }
+}
```

Comment:
not sure how this is used,

Reply:
It's a blank page that's linked to from the overview page.  I put the wrong blank page in, though - it should be "Confirm Your Employer".  I'll swap it.